components/inputs/input-time.js

```diff
@@ -92,13 +123,40 @@ class InputTime extends LitElement {
 
 	render() {
 		const input = html`
-			<input
-				aria-label="${ifDefined(this._getAriaLabel())}"
-				@change="${this._handleChange}"
-				class="d2l-input"
-				?disabled="${this.disabled}"
-				@keypress="${this._handleKeypress}"
-				.value="${this._formattedValue}">
+			<d2l-dropdown>
+				<div
+					id="combo-container"
+					role="combobox"
+					aria-owns="time-menu"
+					class="d2l-dropdown-opener"
+					aria-expanded="false">
+					<input
+						aria-controls="time-menu"
+						aria-label="${ifDefined(this._getAriaLabel())}"
+						@change="${this._handleChange}"
+						class="d2l-input"
+						?disabled="${this.disabled}"
+						@keypress="${this._handleKeypress}"
+						.value="${this._formattedValue}">
+				</div>
+				<d2l-dropdown-menu id="dropdown" no-padding-footer min-width="195">
+					<d2l-menu
+						id="time-menu"
+						role="listbox"
```

Comment:
I'm definitely unsettled on overriding the role given the Aria model.  What are your thoughts on this... using this to get by in the short-term, and then introduce a real `listbox` (refactoring menu and its styles as appropriate)?

Reply:
Wouldn't that involve a new component that we wanted to avoid when we were discussing the radio-menu and the aria role issues earlier?